base_export_async/models/delay_export.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import json
+import operator
+import base64
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models, _
+from odoo.addons.queue_job.job import job
+from odoo.addons.web.controllers.main import CSVExport, ExcelExport
+from odoo.exceptions import Warning
+
+_logger = logging.getLogger(__name__)
+
+
+class DelayExport(models.Model):
+
+    _name = 'delay.export'
+    _description = 'Allow to delay the export'
```

Comment:
add `_auto = False` since there are only model methods?

Reply:
Based on the security comment above, we are going to create records to be able to specify rules. The _auto = False is therefore not necessary anymore.